src/styles/sidebar/sidebar.styl

```diff
@@ -0,0 +1,8 @@
+@import '../common/variables.styl'
+@import '../common/icons.styl'
```

Comment:
Do you think it's necessary to use additional directories inside `styles/` (like `common/`, `header/`, etc) at our scale?

Reply:
I think we need at least common folder inside styles. as for other folders they might be redundant and deserve to be removed